src/rgw/Makefile.am

```diff
@@ -113,8 +113,12 @@ LIBRGW_DEPS += \
 	-lcurl \
 	-lexpat \
 	-lm \
-	-lfcgi \
+	-lfcgi 
+
+if LINUX
+LIBRGW_DEPS += \
```

Comment:
could you be more specific in the commit message? 

- a proper prefix noting the subcomponent the commit is changing
- and the commit message body to document why we made the change


 proper prefix noting the subcomponent this change